packages/logger/__test__/formatters/omitFields.test.js

```diff
@@ -1,11 +1,53 @@
-import { omitFields } from '../../src/formatters/omitFields.js';
+import { omitFields } from '../../src/formatters/';
 
 describe('formatter - omitFields', () => {
   it('should remove fields', () => {
-    const log = { omit1: '12', nested: { omit2: '12' }, name: 'boya' };
+    const log = {
+      omit1: '12',
+      nested: { omit2: '12' },
+      name: 'boya',
+    };
 
     const modifiedLog = omitFields(['omit1', 'nested.omit2'])(log);
 
-    expect(modifiedLog).toEqual({ name: 'boya', nested: {} });
+    expect(modifiedLog).toEqual({
+      nested: {},
+      name: 'boya',
+    });
+  });
+
+  it('should remove fields from specified log path root', () => {
+    const log = {
+      omit: '12',
+      nested: { omit: '12', notOmit: true },
+      name: 'boya',
+    };
+
+    const modifiedLog = omitFields(['omit'], ['nested'])(log);
```

Comment:
this test is confusing...
the main point is to be able to remove `omit` (which is nested), by supplying only omit but not `nested.omit`?
why is it two different arrays?